ddl/ddl_worker_test.go

```diff
@@ -649,6 +649,13 @@ func buildCancelJobTests(firstID int64) []testCancelJob {
 		{act: model.ActionModifyColumn, jobIDs: []int64{firstID + 67}, cancelRetErrs: noErrs, cancelState: model.StateWriteOnly},
 		{act: model.ActionModifyColumn, jobIDs: []int64{firstID + 68}, cancelRetErrs: noErrs, cancelState: model.StateWriteReorganization},
 		{act: model.ActionModifyColumn, jobIDs: []int64{firstID + 69}, cancelRetErrs: []error{admin.ErrCancelFinishedDDLJob}, cancelState: model.StatePublic},
+
+		// for drop indexes
+		{act: model.ActionDropIndexes, jobIDs: []int64{firstID + 70}, cancelRetErrs: []error{admin.ErrCannotCancelDDLJob.GenWithStackByArgs(firstID + 70)}, cancelState: model.StateWriteOnly},
```

Comment:
The job ID needs to start from `firstID + 72`. Otherwise, the corresponding job of the test does not match